browser/android/BUILD.gn

```diff
@@ -10,6 +10,7 @@ source_set("android_browser_process") {
   check_includes = false
 
   sources = [
+    "//brave/browser/android/webcompat_reporter/webcompat_reporter_utils.cc",
```

Comment:
please move this into a separate target and add it using `deps =` block below.

`android_browser_process` has `check_includes = false`, you most likely miss some deps already.

Reply:
removed that change